File: CompSci 1/Unit 16, Iteration - for/forPractice/forPractice.java
public class forPractice
{
    public static void main(String[] args)
    {
        for(int i = 0; i <= 25; i++)
        {
            System.out.print(i + " ");
        }

        System.out.println();
        System.out.println();

        for(int i = 10; i >= 1; i--)
        {
            System.out.print(i + " ");
        }

        System.out.println();
        System.out.println();

        for(int i = 1; i <= 40; i++)
        {
            if((i % 2) != 0)
            {
                System.out.print(i + " ");
            }
        }

        System.out.println();
        System.out.println();

        for(int i = 1; i <= 40; i++)
        {
            if((i % 2) == 0)
            {
                System.out.print(i + " ");
            }
        }

        System.out.println();
        System.out.println();

        for(int i = 0; i <= 200; i += 10)
        {
            System.out.print(i + " ");
        }

        System.out.println();
        System.out.println();

        for(int i = -10; i >= -20; i--)
        {
            System.out.print(i + " ");
        }

        System.out.println();
        System.out.println();

        for(int i = 0; i < 10; i++)
        {
            System.out.println("computer");
        }
    }
}

File: CompSci 1/Unit 16, Iteration - for/forPractice/forPractice.kt
fun main(args: Array<String>) {
    for (i in 0..25) {
        print(i.toString() + " ")
    }

    println()
    println()

    for (i in 10 downTo 1) {
        print(i.toString() + " ")
    }

    println()
    println()

    for (i in 1..40) {
        if (i % 2 != 0) {
            print(i.toString() + " ")
        }
    }

    println()
    println()

    for (i in 1..40) {
        if (i % 2 == 0) {
            print(i.toString() + " ")
        }
    }

    println()
    println()

    run {
        var i = 0
        while (i <= 200) {
            print(i.toString() + " ")
            i += 10
        }
    }

    println()
    println()

    for (i in -10 downTo -20) {
        print(i.toString() + " ")
    }

    println()
    println()

    for (i in 0..9) {
        println("computer")
    }
}